cli.js

```diff
@@ -1926,6 +1926,60 @@ program
         console.log();
     });
 
+program
+    .command('slas:tenant:add')
+    .description('Adds a SLAS tenant to given organisation')
+    .option('--tenant <tenant>', 'the tenant id to add')
+    .option('--shortcode <shortcode>', 'the organizations short code')
+    .option('--merchantname <merchantame>', 'the name given for the tenant')
+    .option('--tenantdescription <tenantdescription>', 'the tenant descriptions')
+    .option('--contact <contact>', 'Contact person to manage tenants')
+    .option('--email <email>', 'Email to contact')
+    .option('-j, --json', 'Formats the output in json')
+    .action(async function(options) {
+
+        var asJson = ( options.json ? options.json : false );
```

Comment:
Do you use the `asJson` as all? If not we should remove it and from the options

Reply:
this is now added